docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for val in [-2, -1.5, -1, -0.5, 0, 0.5, 1, 1.5, 2]:
</pre><p><strong>np.linspace(-2, 2, num=9)</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>